metaci/release/utils.py

```diff
@@ -83,10 +129,98 @@ def send_release_webhook(project_config, release):
     )
     result = response.json()
     if result["success"]:
+        # result["implementationSteps"] = ["1", "2", "3", "4"]
+        #######################################################################
+        # PARSE RESULT HERE FOR IMPLMENTATION ID's when GUS SANDBOX is ready! #
+        #######################################################################
         with transaction.atomic():
+            release.implementation_steps.filter(plan__role="release_deploy").update(
+                external_id=result["implementationSteps"][0]
+            )
+            release.implementation_steps.filter(plan__role="release").update(
+                external_id=result["implementationSteps"][1]
+            )
+            release.implementation_steps.filter(plan__role="push_sandbox").update(
+                external_id=result["implementationSteps"][2]
+            )
+            release.implementation_steps.filter(plan__role="push_production").update(
+                external_id=result["implementationSteps"][3]
+            )
```

Comment:
This part needs to be inside a conditional for the new setting so that we don't try to run it in production yet. It can refer back to the list of implementation steps created above so that we don't need to re-query.